docs/site/assets/css/page.css

```diff
@@ -180,3 +180,8 @@ header#atViewHeader {
 div > header {
   display: none !important;
 }
+
+img {
+  max-width: 100%;
```

Comment:
Sounds like we should use svh & svw

Reply:
https://dev.to/frehner/css-vh-dvh-lvh-svh-and-vw-units-27k4

Reply:
@macchiati what are you seeing instead? When I used `dvh/dvw` from your original proposal, the scale didn't work properly- it was far too wide still.